reservations/models.py

```diff
@@ -98,7 +104,31 @@ class STATE_CHOISE_CONST(object):
 STATE_CHOICES = STATE_CHOISE_CONST()
 
 
+class ReservationQuerySet(models.QuerySet):
+    def total_duration(self):
+        return self.annotate(duration=F("end") - F("begin")).aggregate(
+            total_duration=Sum("duration")
+        )
+
+    def within_application_round_period(self, app_round: ApplicationRound):
+        return self.filter(
```

Comment:
One could maybe wonder if reservation starts before the res period but ends within the res period should be partly part of used capacity, but doesn't really matter, since we don't really need super exact numbers here.